cluster/cluster-resource.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: cluster-resource
```

Comment:
The task name is misleading. Something that implies set kubeconfig, or authenticate to kubernetes might be more appropriate